source/module_hsolver/test/math_kernel_test.cpp

```diff
@@ -542,6 +542,84 @@ TEST_F(TestModuleHsolverMathKernel, gemv_op_gpu)
 
 
 
+TEST_F(TestModuleHsolverMathKernel, matrixSetToAnother_op_gpu)
+{
+  // const std::vector<std::complex<double> > expect_result = {
+  //   {-0.11893203,-0.13492526}, {-0.40314756, 0.07734553}, {0.06892412, 0.14837423}, {0.0,  0.0},
+  //   {0.61158728, -0.45754102}, {-0.54274745,-0.09682102}, {0.30232967, 0.49411249}, {0.0,  0.0}
+  // };
+
+  const std::vector<std::complex<double> > A = {
```

Comment:
We use 4 spaces for indentation; it would be better formatting this file in compliance with other source codes.

Reply:
LGTM